### PR TITLE
BUGBBQ: Improve prerequisites

### PR DESCRIPTION
**Pull request by tomwright01**
Address issue #48, update shell requisite and drop git requisite.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -14,17 +14,18 @@ attendees a strong foundation in the fundamentals of R, and to teach best
...
-packages we find useful, and have been chosen primarily for their 
+are not necessarily the best, nor are they comprehensive, but they are
+packages we find useful, and have been chosen primarily for their
 usability.
 
 > ## Prerequisites {.prereq}
 >
-> Have attended the Shell and Git sessions.
+> Understand that computers store data and instructions (programs, scripts etc.) in files. Files are organised in directories (folders).
+> Know how to access files not in the working directory by specifying the `path`.
```
Comment (mkcor) [inline index.md]: Not sure 'path' should be styled this way, but I can see you want to highlight it (maybe *emphasize*?).
I like your wording but maybe I would keep it at "Understand the filesystem and know how to navigate it from the command line."

File: instructors.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ git remote add data https://github.com/resbaz/r-novice-gapminder-files
 git pull data master
 ~~~
 
+If Git is not being taught as part of the workshop the raw data can be downloaded from the folloing urls:
+[gapminder-FiveYearData](https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/data/gapminder-FiveYearData.csv)
+[gapminder-FiveYearData-Wide](https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/data/gapminder_wide.csv)
+
+attendees will need to use the `File - Save As` dialog in their browser to save the file.
```
Comment (mkcor) [inline instructors.md]: I would say "can use", since it's one possible way.

**Comment by mkcor** [comment]
Ah, trailing whitespaces! :goat:

**Comment by aammd** [comment]
I like all of @tomwright01 's changes and I also like the edits suggested by @mkcor 😄

**Comment by aammd** [comment]
In fact, what I think I'll do is just merge in @tomwright01 's changes, then include those edits suggested by @mkcor . (hope that is OK) Thanks to you both!

**Pull request merged**